docs/introduction/upgrading-to-stencil-three.md

```diff
@@ -406,6 +406,19 @@ $ npm uninstall @types/puppeteer
 
 To see which versions of Puppeteer are supported by Stencil, please see our [support matrix](/docs/support-policy#puppeteer)
 
+## Additional Packages
+
+In order to guarantee that other `@stencil/` packages will continue to function as expected,
+it's recommended that a project that uses any of the packages listed below updates to the listed minimum package version.
+
+| Package                          | Minimum Package Version | GitHub                                                            | Documentation                                               |
+|----------------------------------|-------------------------|-------------------------------------------------------------------|-------------------------------------------------------------|
+| `@stencil-angular-output-target` | 0.5.0                   | [GitHub](https://github.com/ionic-team/stencil-ds-output-targets) | [Stencil Doc Site](/docs/angular)                           |
```

Comment:
should we maybe link to the releases here? I know that's kind of fiddly, but something like this:

```suggestion
| `@stencil-angular-output-target` | [0.5.0](https://github.com/ionic-team/stencil-ds-output-targets/releases/tag/%40stencil%2Fangular-output-target%400.5.0)                   | [GitHub](https://github.com/ionic-team/stencil-ds-output-targets) | [Stencil Doc Site](/docs/angular)                           |
```

Just to make is easy to see if there's a breaking change that the upgrader might need to account for

Reply:
👍 done in https://github.com/ionic-team/stencil-site/pull/1023/commits/4a82689f98d3cd0f5bafa184e1cd9a52d787feef